packages/graphql-server/src/plugins/useRedwoodRealtime.ts

```diff
@@ -19,35 +19,162 @@ export const liveDirectiveTypeDefs = print(
   astFromDirective(GraphQLLiveDirective)
 )
 
+export type LiveQueryStorageMechanism =
+  | RedisLiveQueryStore
+  | InMemoryLiveQueryStore
+
 export type RedwoodRealtimeOptions = {
```

Comment:
The realtime options are more capable now to config in-memory or redis backed stores.

And since it can create them, it extends the context with them so the app code doesn't need to.